portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/SocialBookmarks.path

```diff
@@ -12,7 +12,7 @@
 <tbody>
 <tr>
 	<td>SOCIAL_BOOKMARK_ICON</td>
-	<td>//li[contains(@class,'taglib-social-bookmark-${key_socialBookmark}')]/a</td>
+	<td>//li[contains(@class,'taglib-social-bookmark-${key_socialBookmark}')]/*/a</td>
```

Comment:
Hi @john-co,

Would like some help updating this path correctly, would like to have something like `li/whatever-element-or-none/a`

Not sure if the way I updated is exactly what I wanted to achieve.

Thanks.

Reply:
Hi @georgel-pop-lr, 

I have downloaded and checked your bundle and yes, the updated path `<td>//li[contains(@class,'taglib-social-bookmark-${key_socialBookmark}')]/*/a</td>` is correct for the social bookmarks.

So you can move forward with it 😉

Reply:
Hi @carloslancha,

Not sure how to forward this if there are still not related tests failing.

Could you help?

Regards.